packages/react-native-bridge/android/react-native-bridge/build.gradle

```diff
@@ -65,20 +65,20 @@ dependencies {
 
     implementation "com.facebook.react:react-native:$rnVersion"
     implementation "org.wordpress-mobile:react-native-svg:${extractPackageVersion(packageJson, 'react-native-svg', 'dependencies')}"
-    implementation "com.github.wordpress-mobile:react-native-video:${extractPackageVersion(packageJson, 'react-native-video', 'dependencies')}"
-    implementation "com.github.wordpress-mobile:react-native-linear-gradient:${extractPackageVersion(packageJson, 'react-native-linear-gradient', 'dependencies')}"
-    implementation "com.github.wordpress-mobile:react-native-slider:${extractPackageVersion(packageJson, '@react-native-community/slider', 'dependencies')}"
+    implementation "com.github.wordpress-mobile:react-native-video:update~android-12-SNAPSHOT"
+    implementation "com.github.wordpress-mobile:react-native-linear-gradient:update~android-12-SNAPSHOT"
+    implementation "com.github.wordpress-mobile:react-native-slider:update~android-12-SNAPSHOT"
     implementation "org.wordpress-mobile:react-native-get-random-values:${extractPackageVersion(packageJson, 'react-native-get-random-values', 'dependencies')}"
     implementation "org.wordpress-mobile:react-native-masked-view:${extractPackageVersion(packageJson, '@react-native-masked-view/masked-view', 'dependencies')}"
     implementation "org.wordpress-mobile:react-native-screens:${extractPackageVersion(packageJson, 'react-native-screens', 'dependencies')}"
     implementation "org.wordpress-mobile:react-native-safe-area-context:${extractPackageVersion(packageJson, 'react-native-safe-area-context', 'dependencies')}"
-    implementation "com.github.wordpress-mobile:react-native-reanimated:${extractPackageVersion(packageJson, 'react-native-reanimated', 'dependencies')}"
-    implementation "com.github.wordpress-mobile:react-native-prompt-android:${extractPackageVersion(packageJson, 'react-native-prompt-android', 'dependencies')}"
+    implementation "com.github.wordpress-mobile:react-native-reanimated:update~android-12-1.9.0-g0ae9a17"
+    implementation "com.github.wordpress-mobile:react-native-prompt-android:update~android-12-SNAPSHOT"
     implementation "org.wordpress-mobile:react-native-webview:${extractPackageVersion(packageJson, 'react-native-webview', 'dependencies')}"
     implementation "org.wordpress-mobile:react-native-clipboard:${extractPackageVersion(packageJson, '@react-native-clipboard/clipboard', 'dependencies')}"
     implementation "org.wordpress-mobile:react-native-fast-image:${extractPackageVersion(packageJson, 'react-native-fast-image', 'dependencies')}"
 
-    implementation("com.github.wordpress-mobile:react-native-gesture-handler:${extractPackageVersion(packageJson, 'react-native-gesture-handler', 'dependencies')}", {
+    implementation("com.github.wordpress-mobile:react-native-gesture-handler:update~android-12-SNAPSHOT", {
```

Comment:
The reference of these dependencies will be updated once their associated PRs, which also update the Android API version are merged:
- [react-native-gesture-handler](https://github.com/wordpress-mobile/react-native-gesture-handler/pull/23)
- [react-native-reanimated](https://github.com/wordpress-mobile/react-native-reanimated/pull/19)
- [react-native-prompt-android](https://github.com/wordpress-mobile/react-native-prompt-android/pull/9)
- [react-native-linear-gradient](https://github.com/wordpress-mobile/react-native-linear-gradient/pull/7)
- [react-native-slider](https://github.com/wordpress-mobile/react-native-slider/pull/12)
- [react-native-video](https://github.com/wordpress-mobile/react-native-video/pull/20)

Reply:
Solving this as all dependencies have been updated.